tap_chargebee/streams/base.py

```diff
@@ -235,36 +234,52 @@ def sync_data(self, child_sync=False):
                     to_write.append(coupon)
 
 
-            with singer.metrics.record_counter(endpoint=table) as ctr:
-                singer.write_records(table, to_write)
+            if not sync_data_for_child_stream:
```

Comment:
Added this if so we can have the default behavior as default and indented lines below it.

Reply:
This is done this way so we still have the access of the done value update while iterating through child stream